utils/spdx/src/main/resources/declared-license-mapping.yml

```diff
@@ -142,6 +142,7 @@
 "BSD 2-clause \\\"Simplified\\\" or \\\"FreeBSD\\\"\n                License": BSD-2-Clause OR BSD-2-Clause-Views
 "BSD 3 Clause": BSD-3-Clause
 "BSD 3": BSD-3-Clause
+"BSD 3-Clause License or Apache License, Version 2.0": Apache-2.0 OR BSD-3-Clause
```

Comment:
Should we maintain the original order for easier understand of the mapping (like we do in line 30)?